src/driver.rs

```diff
@@ -53,38 +102,63 @@ impl Driver {
     }
 
     /// Returns the number of registered drivers.
+    ///
+    /// # Example
+    ///
+    /// ```rust, no_run
+    /// use gdal::Driver;
+    /// println!("{} drivers are registered", Driver::count());
+    /// ```
+    /// ```text
+    /// 203 drivers are registered
+    /// ```
     pub fn count() -> usize {
         _register_drivers();
         let count = unsafe { gdal_sys::GDALGetDriverCount() };
         count.try_into().unwrap()
     }
 
-    /// Creates a new Driver object by wrapping a C pointer
+    /// Return the short name of a driver.
     ///
-    /// # Safety
-    /// This method operates on a raw C pointer
-    pub unsafe fn from_c_driver(c_driver: GDALDriverH) -> Driver {
-        Driver { c_driver }
-    }
-
-    /// Returns the wrapped C pointer
+    /// For the GeoTIFF driver, this is “GTiff”
     ///
-    /// # Safety
-    /// This method returns a raw C pointer
-    pub unsafe fn c_driver(&self) -> GDALDriverH {
-        self.c_driver
-    }
-
+    /// See also: [`long_name`](Self::long_name).
     pub fn short_name(&self) -> String {
         let rv = unsafe { gdal_sys::GDALGetDriverShortName(self.c_driver) };
         _string(rv)
     }
 
+    /// Return the short name of a driver.
+    ///
+    /// For the GeoTIFF driver, this is “GeoTIFF”
+    ///
+    /// See also: [`short_name`](Self::short_name`).
     pub fn long_name(&self) -> String {
         let rv = unsafe { gdal_sys::GDALGetDriverLongName(self.c_driver) };
         _string(rv)
     }
 
+    /// Create a new dataset of size (`size_x`, `size_y`) and `bands` band count,
+    /// and [`u8`] as the cell data type.
+    ///
+    /// To specify an alternative data type (e.g. [`f32`]), use [`create_with_band_type`](Self::create_with_band_type).
```

Comment:
TBH, I don't think `u8` deserve special-casing.

Reply:
@lnicola Is this a comment about the markup I used, or about the existence of a special case `create` function, specifically for `u8`?

Reply:
About the existence of the function, there's nothing we can do about it in this PR.

Reply:
Yeh, I completely agree. It was actually confusing to me when I first came upon it.